### PR TITLE
Dev/obj track set pybind imports

### PR DESCRIPTION
**Pull request by as6520**
Fix pybind include and simple test case for object track set and state

### CODE REVIEW

File: vital/bindings/python/vital/tests/test_object_track_state.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        nose.tools.assert_equal(ts.time_usec, 14691234578)
+
+
+    def test_detection(self):
+        do = self._create_detected_object()
+        ts = ObjectTrackState(0, 0, do)
+        nose.tools.assert_equal(ts.detection, do)
+
+
+'''
```
Comment (Purg) [inline vital/bindings/python/vital/tests/test_object_track_state.py]: Are these commented tests relevant?  If not we should remove this.

File: vital/bindings/python/vital/tests/test_object_track_set.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+import nose.tools
+import numpy
+
+from six.moves import range
+
+from vital.types import ObjectTrackSet, ObjectTrackState, BoundingBox, \
+        DetectedObjectType, DetectedObject, Track
+
+
+class TestObjectTrackState (unittest.TestCase):
```
Comment (Purg) [inline vital/bindings/python/vital/tests/test_object_track_set.py]: I think this should probably be ``TestObjectTrackSet`` instead of ``State``.

File: vital/bindings/python/vital/tests/test_object_track_state.py [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+from vital.types import ObjectTrackState, BoundingBox, DetectedObjectType, \
+        DetectedObject
+
+
+class TestObjectTrackState (unittest.TestCase):
+
+    def _create_detected_object(self):
+        bbox = BoundingBox(10, 10, 20, 20)
+        dot  = DetectedObjectType("test", 0.4)
+        do = DetectedObject(bbox, 0.4, dot)
```
Comment (Purg) [inline vital/bindings/python/vital/tests/test_object_track_state.py]: Missing ``return`` from this function, which currently returns ``None`` by default.  The tests here involving detected objects are erroneously passing because they are adding and testing the return of ``None``.
